releasenotes-builder/src/test/resources/com/github/checkstyle/githubPageBreakingCompatibility.txt

```diff
@@ -1,4 +1,4 @@
-Checkstyle 1.0.0 - https://checkstyle.org/releasenotes.html#Release_1.0.0
+Checkstyle 1.0.1 - https://checkstyle.org/releasenotes.html#Release_1.0.1

```

Comment:
why notes with breacking changes is patch ?
if we do not validate version, lets still keep version reasonable.

Reply:
Completely missed this detail. `New` and `breaking changes` are staying minor. Done.